src/lib/objects/split/chat-messages/settle-reminder.svelte

```diff
@@ -72,7 +72,8 @@
 			</Container>
 		</div>
 	</ChatMessage>
-{:else if paid || owedAmount > 0n}
+{:else if paid || owedAmount <= 0n}
```

Comment:
`paid` is always false in this file, can it be removed?

Reply:
nice catch, removed thanks!